build.sbt

```diff
@@ -7,10 +7,21 @@ import sbt.Keys._
 import sbt.librarymanagement.ModuleID
 import sbt.nio.Keys._
 
-lazy val scala212 = "2.12.8"
-lazy val scala211 = "2.11.12"
+lazy val scalaNew = "2.12.8"
+lazy val scalaOld = "2.11.12"
 
-lazy val sparkVersion = sys.env.getOrElse("SPARK_VERSION", "3.0.0")
+lazy val sparkNew = "3.0.0"
```

Comment:
likewise, I think we should use `spark3` and `spark2`

Reply:
reverted